From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Sat, 15 Feb 2020 08:20:40 +0100
Subject: Restore GPU bug blacklist for AImageReader on ARM CPUs

Restore the AImageReader blacklist for ARM/ARM64 devices which causes
crashes on Android 9 and 10 (at different code locations).

See discussion at https://github.com/bromite/bromite/issues/445
---
 gpu/config/gpu_driver_bug_list.json | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/gpu/config/gpu_driver_bug_list.json b/gpu/config/gpu_driver_bug_list.json
--- a/gpu/config/gpu_driver_bug_list.json
+++ b/gpu/config/gpu_driver_bug_list.json
@@ -3537,6 +3537,22 @@
         "dont_delete_source_texture_for_egl_image"
       ]
     },
+    {
+      "id":335,
+      "cr_bugs": [1051705],
+      "description": "Disable AImageReader on ARM GPUs",
+      "os": {
+        "type": "android",
+        "version": {
+          "op": "<",
+          "value": "10"
+        }
+      },
+      "gl_vendor": "ARM.*",
+      "features": [
+        "disable_aimagereader"
+      ]
+    },
     {
       "id": 336,
       "cr_bugs": [625785],
-- 
2.17.1

